### PR TITLE
Fixed broken links in Quantum Machine learning course 

### PR DESCRIPTION
**Pull request by divshacker**
## Changes
fixes Qiskit-community/qiskit-textbook#1379 #713 

## Implementation details
As mentioned in above linked issue, There are 4 points mentioned and I have fixed 2 of them (2nd and 3rd)

### CODE REVIEW

**Comment by divshacker** [comment]
For first one I need the correct link and the given link in the issue is not correct.

**Review by frankharkins — DISMISSED** [review]
Thank you!

I updated the slack links, and added a `https://` to the quantum lab links.

File: notebooks/intro/project.ipynb [hunk 1]
```diff
@@ -122,18 +122,18 @@
...
-    "After you’ve put in the time and effort, share your work with others on the [`#textbook-projects`](https://join.slack.com/share/enQtMjcxMjMyNzczMzg1Ni02ZDQ5NTAwZGIwMzJhOTQzMzUzZDNjOTg3ZTM3YzY2ZTRmZTRiMWMwZjZmMTczYTEyMTE0NThiMGNiOTFiOWQ0) channel. If you maintain your GitHub profile and GitHub pages web page, you can use this to evidence your knowledge and experience to others (e.g. in applications or interviews)."
+    "After you’ve put in the time and effort, share your work with others on the [`#textbook-projects`](https://qiskit.slack.com/archives/C028AGU6KDX) channel. If you maintain your GitHub profile and GitHub pages web page, you can use this to evidence your knowledge and experience to others (e.g. in applications or interviews)."
    ]
   }
  ],
  "metadata": {
   "celltoolbar": "Edit Metadata",
   "kernelspec": {
-   "display_name": "Python 3",
+   "display_name": "Python 3 (ipykernel)",
```
Comment (Tansito) [inline notebooks/intro/project.ipynb:136]: Thanks for the contribution @divshacker ! Could you remove this change from the PR? We would like to maintain the changes only related with the new links 🙏

File: notebooks/intro/project.ipynb [hunk 2]
```diff
@@ -147,7 +147,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.7.7"
+   "version": "3.9.10"
```
Comment (Tansito) [inline notebooks/intro/project.ipynb:150]: Same as my previous comment.

Comment (Tansito) [inline notebooks/intro/project.ipynb:150]: Same as my previous comment.

File: notebooks/quantum-machine-learning/kernel.ipynb [hunk 3]
```diff
@@ -9434,7 +9434,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.9.8"
+   "version": "3.8.10"
```
Comment (Tansito) [inline notebooks/quantum-machine-learning/kernel.ipynb:9437]: Same as my previous comment.

Comment (Tansito) [inline notebooks/quantum-machine-learning/kernel.ipynb:9437]: Same as my previous comment.

File: notebooks/quantum-machine-learning/project.ipynb [hunk 4]
```diff
@@ -139,7 +139,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.9.8"
+   "version": "3.9.10"
```
Comment (Tansito) [inline notebooks/quantum-machine-learning/project.ipynb:142]: Same as my previous comment.

Comment (Tansito) [inline notebooks/quantum-machine-learning/project.ipynb:142]: Same as my previous comment.

File: notebooks/quantum-machine-learning/training.ipynb [hunk 5]
```diff
@@ -16391,7 +16391,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.9.8"
+   "version": "3.8.10"
```
Comment (Tansito) [inline notebooks/quantum-machine-learning/training.ipynb:16394]: And last one change if you can revert it.

Comment (Tansito) [inline notebooks/quantum-machine-learning/training.ipynb:16394]: And last one change if you can revert it.

**Review by Tansito — CHANGES_REQUESTED** [review]

**Comment by divshacker** [comment]
Hey @Tansito I have fixed all the unnecessary changes. 
Let me know if there is anything else required
Thanks

**Review by Tansito — APPROVED** [review]

**Pull request merged**